pages/draw.md

```diff
@@ -111,6 +111,13 @@ text:
     anchor: bottom-left # the label will use `align: right` by default
 ```
 
+####`flat` 
+[[ES-only](https://github.com/tangrams/tangram-es)]
```

Comment:
It's used with the same meaning in Google Maps: https://developers.google.com/maps/documentation/android-api/marker#flatten_a_marker